### PR TITLE
Increase default multipart size to 128MiB

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

**Comment by ebozduman** [comment]
More references to min part as 64MiB in the following files and line numbers (almost all comments) :

| FileName | LineNumber |
| ------------ | ----------------- |
| api-put-object-common.go | 67 |
| api-put-object-streaming.go | 78 |
| api-put-object.go | 127, 129 , 170|
| ?? api-stat.go ?? | ?? 139 ?? | 
| core_test.go | 39, 44 |
| API.md | 531(3), 872(3) |
| ?? functional_tests.go ?? | ?? 834(2), 838 ?? |

**Comment by kannappanr** [comment]
PTAL at Travis failures

File: api_unit_test.go [hunk 1]
```diff
@@ -120,10 +120,10 @@ func TestPartSize(t *testing.T) {
 	if err != nil {
 		t.Fatal("Error: ", err)
 	}
-	if totalPartsCount != 79 {
+	if totalPartsCount != 40 {
 		t.Fatalf("Error: expecting total parts count of 79: got %v instead", totalPartsCount)
```
Comment (kannappanr) [inline api_unit_test.go]: `Error: expecting total parts count of 79` should be `Error: expecting total parts count of 40`

File: api_unit_test.go [hunk 2]
```diff
@@ -120,10 +120,10 @@ func TestPartSize(t *testing.T) {
...
 	}
-	if partSize != 67108864 {
+	if partSize != 134217728 {
 		t.Fatalf("Error: expecting part size of 67108864: got %v instead", partSize)
```
Comment (kannappanr) [inline api_unit_test.go]: `expecting part size of 67108864` should be `expecting part size of 134217728`

**Pull request merged**